tsdb_synth/track.json

```diff
@@ -0,0 +1,33 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "synthetic time series dataset for tsid hashing benchmarks",
+  "indices": [
+    {
+      "name": "tsdb_synth",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "tsdb_synth",
+      "base-url": "https://storage.googleapis.com/rally-tracks/tsdb_synth/",
```

Comment:
Could this be https://rally-tracks.elastic.co/tsdb_synth/ ? They are the same place at the moment, but https://rally-tracks.elastic.co/tsdb_synth/ goes through the cdn

Reply:
Ok...so going through theCDN was probably the cause of the issue about the file not being there I guess...just a result of the delay in distributing the file across the CDN probably.